src/DFG.h

```diff
@@ -65,6 +66,7 @@ class DFG {
     void combinePhiAdd();
     void combine(string, string);
     void combineForIter(list<string>*);
+    void combineForUnroll(list<string>*); //combineForUnroll
```

Comment:
Can you put the comments above the function declaration and make it more detailed?